larq_compute_engine/mlir/transforms/fuse_padding.td

```diff
@@ -43,8 +43,7 @@ def : Pat<(TFL_Conv2DOp:$conv_output
           [(HasOneUse $pad_output),
            (NoBatchAndChannelPadding $paddings),
            (SamePaddingHeight $paddings, $input, $conv_output, $stride_h),
-           (SamePaddingWidth $paddings, $input, $conv_output, $stride_w)],
-          (addBenefit 100)>;
```

Comment:
There is now an extra argument _before_ the benefit-argument. So we could either insert a `[], ` here before `addBenefit`, or just remove it. @lgeiger do you know if we depend on these benefit values? A lot of them seem arbitrary to me, but some seem to be chosen specifically.

Reply:
I looked through the patterns again, and I added back one of the `addBenefit`s and left all the others out.

The one I added back was in order to give priority to `GreaterEqual(x, 0) -> Quantize` pattern over the more generic `GreaterEqual(x, threshold) -> Quantize(x - threshold)` pattern. This might avoid a 'subtract 0' operation.
For all other patterns, I don't think the benefit value really mattered: you could first convert a `ste.sign(x)` to `Quantize(Dequantize(x))` or first convert a Conv to a Bconv; that order does not matter. (Besides, it only applies to patterns _within_ one pass)